core/iwasm/compilation/aot_compiler.c

```diff
@@ -2853,42 +2696,33 @@ aot_compile_wasm(AOTCompContext *comp_ctx)
         }
     }
 
-#if WASM_ENABLE_LAZY_JIT != 0
-    orc_main_dylib = LLVMOrcLLJITGetMainJITDylib(comp_ctx->orc_lazyjit);
+#if WASM_ENABLE_JIT != 0
```

Comment:
How about removing the macro control, and replace it as
```
    if (comp_ctx->is_jit_mode) { ... }
```
Seems after changing, there is no `WASM_ENABLE_JIT` macro controls in this folder.